Source/Core/Core/CoreParameter.cpp

```diff
@@ -373,6 +373,14 @@ void SCoreStartupParameter::CheckMemcardPath(std::string& memcardPath, std::stri
 	}
 }
 
+IVolume::ELanguage SCoreStartupParameter::GetCurrentLanguage(bool wii)
+{
+	if (wii)
+		return (IVolume::ELanguage)SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.LNG");
```

Comment:
Blindly casting from u8 to ELanguage seems like a bad idea: we have no idea whether the value is actually valid.